src/Stratis.Bitcoin.Features.SmartContracts.Tests/SmartContractExecutorTests.cs

```diff
@@ -117,7 +117,7 @@ public void SmartContractExecutor_CallContract_DoesNotExist_Refund()
             Assert.IsType<SmartContractDoesNotExistException>(result.Exception);
         }
 
-        [Fact]
+        [Fact(Skip = "The assumptions made in this test are no longer applicable")]
```

Comment:
We should address what the assumptions are: specifically that Exception is no longer an allowed type.

Could be good to add a TODO: to test what this was testing - which we could still do via an Assert rather than an Exception in the constructor. 

Don't re-implement the test yet - constructor behaviour will be specced out soon anyhow.